packages/flutter/lib/src/widgets/animated_cross_fade.dart

```diff
@@ -368,14 +368,12 @@ class _AnimatedCrossFadeState extends State<AnimatedCrossFade> with TickerProvid
         ),
       ),
     );
-    return ClipRect(
-      child: AnimatedSize(
-        alignment: widget.alignment,
-        duration: widget.duration,
-        reverseDuration: widget.reverseDuration,
-        curve: widget.sizeCurve,
-        child: widget.layoutBuilder(topChild, topKey, bottomChild, bottomKey),
-      ),
+    return AnimatedSize(
```

Comment:
This seems dangerous. If the content is indeed bigger, this may cause a regression

Reply:
I think this ClipRect here is undesired behavior but i would understand concerns about changing default behaviour. Making this optional while leaving default behavior as is may be a better idea. Let me prepare a commit for it.

Reply:
It might be better to expose clipBehavior as a property so folks can disable/enable/configure as they like.